doc/13-repack.md

```diff
@@ -0,0 +1,30 @@
+# Repacking your application in a new phar
+
+You have created a castor application, with many tasks, and you want to
+distribute it as a single phar file? Castor can help you with that.
+
+In your project, install castor as a dependency:
+
+```bash
+composer require castor/castor
+```
+
+You'll also need
+[box](https://github.com/box-project/box/blob/main/doc/installation.md#installation)
+to create the phar.
```

Comment:
You should add a note about adding the box phar into the PATH. So it's found, I took some time understanding it needs to be that way (tried installing through composer globally, downloaded the phar in the same directory as the code I'm repacking ...)